packages/entity-detail/src/components/DetailForm/SaveButton.js

```diff
@@ -1,28 +1,61 @@
-import {Button} from 'tocco-ui'
+import {Button, Popover} from 'tocco-ui'
 import React from 'react'
 import {intlShape} from 'react-intl'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
 import modes from '../../util/modes'
+import ErrorBox from '../ErrorBox'
 
-const SaveButton = ({submitting, mode, intl}) => {
+const StyledButton = styled(Button)`
+  ${({hasErrors, theme}) => hasErrors && `
+   background-color: ${theme.colors.signal.warning.text};
```

Comment:
`text` color as background color? won't this mess things up when someone changes that color in the theme? I guess I wouldn't expect it to be used as background color somewhere.